src/Scripting/Core/ObjectFormatter.Formatter.cs

```diff
@@ -344,18 +346,22 @@ private void FormatObjectMembersRecursive(List<FormattedMember> result, object o
             {
                 Debug.Assert(obj != null);
 
-                var type = obj.GetType();
-                var fields = type.GetFields(Ref.BindingFlags.Instance | Ref.BindingFlags.Public | Ref.BindingFlags.NonPublic);
-                var properties = type.GetProperties(Ref.BindingFlags.Instance | Ref.BindingFlags.Public | Ref.BindingFlags.NonPublic);
-                var members = new List<Ref.MemberInfo>(fields.Length + properties.Length);
-                members.AddRange(fields);
-                members.AddRange(properties);
+                var members = new List<MemberInfo>();
```

Comment:
It is but I'd stick with List. There isn't much difference between allocating a new ArrayBuilder and List. Using pooled builder would require try-finally and other changes in the logic.